autoload/maktaba/json.vim

```diff
@@ -1,8 +1,7 @@
 " Neovim support covered in https://github.com/neovim/neovim/issues/3417.
 " NOTE: Avoids using pre-1430 support for consistent Infinity/NaN.
-let s:HAS_NATIVE_JSON =
-    \ !has('nvim') &&
-    \ v:version > 704 || v:version == 704 && has('patch1430')
+let s:HAS_NATIVE_JSON = exists('*json_decode') && exists('v:null') &&
```

Comment:
IIRC the sentinels were added after the initial implementation but before patch 1430 (on neovim if not on vim). With both independent commit histories to worry about and the possibility of taking patch 1430 but missing earlier patches, I figured let's just check that explicitly.